src/test/java/net/minecraftforge/debug/misc/GameTestTest.java

```diff
@@ -0,0 +1,72 @@
+package net.minecraftforge.debug.misc;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.gametest.framework.GameTest;
+import net.minecraft.gametest.framework.GameTestGenerator;
+import net.minecraft.gametest.framework.GameTestHelper;
+import net.minecraft.gametest.framework.GameTestRegistry;
+import net.minecraft.gametest.framework.TestFunction;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.level.block.Blocks;
+import net.minecraft.world.level.block.Rotation;
+import net.minecraftforge.fml.common.Mod;
+
+import java.util.List;
+
+@Mod(GameTestTest.MODID)
+public class GameTestTest {
```

Comment:
Brackets in this class should be on new lines just like how it is done in other test mods and forge code. (Note: that **_does not_** mean **just** this _specific_ bracket I am commenting on).